content-src/components/DiscoveryStreamComponents/DSMessage/_DSMessage.scss

```diff
@@ -43,3 +43,53 @@
     }
   }
 }
+
+.ds-chevron-link {
```

Comment:
Any reason css is combined with DSMessage and not as _DSTextPromo.scss? I believe this assumes DSMessage loaded when showing TextPromo